cockatrice/src/dlg_connect.cpp

```diff
@@ -69,6 +71,19 @@ DlgConnect::DlgConnect(QWidget *parent)
 
     connect(savePasswordCheckBox, SIGNAL(stateChanged(int)), this, SLOT(passwordSaved(int)));
 
+	btnForgotPassword = new QPushButton(tr("Forgot password"));
+	btnForgotPassword->setFixedWidth(100);
```

Comment:
Yeah this was left over from a previous configuration of the dialog box and was missed when the buttons were redesigned.